pkg/registry/networking/networkpolicy/storage/storage.go

```diff
@@ -61,3 +61,11 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"netpol"}
 }
+
+//Implement CategoriesProvider
+var _ rest.CategoriesProvider = &Rest{}
+
+// Categories implements the CategoriesProvider interface. Returns a list of categories a resource is part of.
+func (r *REST) Categories() []string{
+	return []string("networkpolicies")
```

Comment:
```suggestion
	return []string{"all"}
```

@soltysh is `all` here?

Reply:
in this [comment](https://github.com/kubernetes/kubernetes/issues/106176#issuecomment-963088769) @soltysh mentioned that to use `all` is wrong.

Reply:
what he says, but you have the format wrong, you need to use brackets not parenthesis
```
return []string{"networkpolicies"}
```

Reply:
good catch!

Reply:
I am a little confused.  That comment says "don't add it to 'all'" which I understand.  What's the point of adding a "networkpolicies" category?  If we want a category, we want more than one resource in it, right?